### PR TITLE
Add DietPi support

### LINKED ISSUE #1705: Distro request | DietPi
## Description
Could you add DietPi? https://dietpi.com

[ASCII Art](https://ascii-generator.site/r/xtrq7S/)

### PR DESCRIPTION
**Pull request by nonnorm**
## Description

Adds the DietPi linux "distro".

Resolves #1705

## Features
Add DietPi versoning and ASCII logo

## Issues
Nothing except for the outstanding one that APT can take a while.

## TODO
Nothing

### CODE REVIEW

**Comment by ravenclaw900** [comment]
Screen Shot 2021-03-21 at 1 01 10 PM

**Comment by koutsie** [comment]
LGTM!

**Comment by dylanaraps** [comment]
Existing detection methods do not work?

File: neofetch [hunk 1]
```diff
@@ -999,6 +999,14 @@ get_distro() {
...
                 esac
 
+            elif [[ -f /boot/dietpi/.version ]]; then
+                . /boot/dietpi/.version
+                case $distro_shorthand in
+                    on) distro="DietPi v$G_DIETPI_VERSION_CORE.$G_DIETPI_VERSION_SUB.$G_DIETPI_VERSION_RC" ;;
+                    tiny) distro='DietPi' ;;
+                    *) distro="DietPi v$G_DIETPI_VERSION_CORE.$G_DIETPI_VERSION_SUB.$G_DIETPI_VERSION_RC ($G_GITOWNER/$G_GITBRANCH)"
+                esac
+
```
Comment (dylanaraps) [inline neofetch:1009]: The existing detection methods don't work?

Comment (ravenclaw900) [reply neofetch:1009]: DietPi is based on Debian/Raspbian, and shows up as such.  It's really just a bunch of scripts and optimizations on top of that, and not a full distribution.

Comment (dylanaraps) [reply neofetch:1009]: If it's not a full distribution, it shouldn't be added to neofetch (line has to be drawn somewhere). Thanks for the PR.

**Comment by hykilpikonna** [comment]
Since DietPi can be detected with existing distro detection methods now, I can merge this.

**Comment by hykilpikonna** [comment]
Thank you for your contribution! 

This PR is [merged into hyfetch](https://github.com/hykilpikonna/hyfetch/commit/02b6e9d7) since this repo (dylanaraps/neofetch) seems no longer maintained.

[HyFetch](https://github.com/hykilpikonna/hyfetch) is a fork of neofetch with LGBTQ pride flags, but the repo also maintains an updated version of the original neofetch, addressing many pull requests that are not merged in the original repo.

Read the ["Running Updated Original Neofetch" section](https://github.com/hykilpikonna/hyfetch#running-updated-original-neofetch) for more info!

**Comment by nonnorm** [comment]
Thanks!

**Pull request closed without merging**